flexget/plugins/input/json.py

```diff
@@ -0,0 +1,137 @@
+"""Plugin for json file."""
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+import dateutil.parser as parser
+
+import re
+import os
+import logging
+import glob
+
+from flexget import plugin
+from flexget.utils import json
+from flexget.entry import Entry
+from flexget.event import event
+
+log = logging.getLogger('json')
+
+
+class Json(object):
```

Comment:
no need to inherit from `object` anymore

Reply:
I'm assuming that I just delete `object` from between the parenthesis?

Reply:
you don't need the parenthesis at all

Reply:
got it